bigframes/functions/_function_session.py

```diff
@@ -129,6 +130,13 @@ def remote_function(
         .. deprecated:: 0.0.1
         This is an internal method. Please use :func:`bigframes.pandas.remote_function` instead.
 
+        .. warning::
+            To use remote functions with Bigframes 2.0 and onwards, please set an
+            explicit user-managed cloud_function_service_account or explicitly set
+            cloud_function_service_account to `"default"`.
```

Comment:
Let's indicate that we prefer an explicit service account.

```suggestion
            To use remote functions with Bigframes 2.0 and onwards, please (preferred)
            set an explicit user-managed ``cloud_function_service_account`` or (discouraged)
            set ``cloud_function_service_account`` to use the Compute Engine service account
            by setting it to `"default"`.
```